doc/api/corepack.md

```diff
@@ -12,10 +12,7 @@ added:
 
 > Stability: 1 - Experimental
```

Comment:
change stability to deprecated
```suggestion
> Stability: 0 - Deprecated
```

That's the closest one from https://github.com/nodejs/node/blob/f692878dec6354c0a82241f224906981861bc840/doc/api/documentation.md